### ISSUE TITLE
Support Graphite's yMax (for both left and right y-axes)

### ISSUE THREAD

**Issue by nhooey** [jondot/graphene#12]
You could either grab the `yMax` parameter from the URL, or support your own `min` and `max` parameters specified in the `TimeSeries` widget type.

**Comment by davidmarquis** [comment]
Yeah Graphite does not do anything with the yMax and yMin parameters when format=json is requested, so it's up to the view (in our case D3.js) to make that call. The `min` and `max` params on the TimeSeries seem like a good option! +1
